Description: 2**31 hashes is too many
 If no blocks are generated for 20 minutes, then difficulty is temporarily
 lowered to 1.  This patch allows difficulty as low as 0.001 so that mining
 blocks on my tegra2 at 140000*2 hashespersec does not take an hour on average.
 Only after the network is down do I expect to see it really go that low, but I
 don't really know what 0.5 difficulty is compared to 1 either.  I would have
 to guess that 0.5 is half as difficult as 1 (eg 1/2 hour on average.)
 .
 bitcoin (0.7.2-4) unstable; urgency=low
 .
   [ Kingdon Barrett ]
   * Allow difficulty as low as 0.001

--- bitcoin-0.7.2.orig/src/rpcblockchain.cpp
+++ bitcoin-0.7.2/src/rpcblockchain.cpp
@@ -16,7 +16,7 @@ double GetDifficulty(const CBlockIndex*
     if (blockindex == NULL)
     {
         if (pindexBest == NULL)
-            return 1.0;
+            return 0.001;
         else
             blockindex = pindexBest;
     }
